### PR TITLE
Added Darrell's info under teammate C

### PR DESCRIPTION
**Pull request by Open-Fuse**
Fixed merge conflicts from ma and Ivy working in the same area
Added myself under teammate C
Realized Debbie's info got removed somewhere down the line and re-added that as well.
Also fixed the body tag in the beginning

### CODE REVIEW

**Review by dayhahm — APPROVED** [review]
Favorite basketball team?

File: src/main/webapp/aboutus.html [hunk 1]
```diff
@@ -21,7 +18,7 @@
     <title>About Us</title>
     <link rel="stylesheet" href="/css/main.css">
   </head>
-  </head>
```
Comment (NickChapman) [inline src/main/webapp/aboutus.html:24]: Did we pickup a bonus `</head>` at some point?

Comment (Open-Fuse) [reply src/main/webapp/aboutus.html:24]: We had an extra `</head>` closing tag and no `<body>` opening tag,

File: src/main/webapp/aboutus.html [hunk 2]
```diff
@@ -41,17 +38,11 @@ <h2>Ivy Zhang</h2>
...
-      <li>Aspirational Hobby: </li>
-      <li>Ask me About: </li>
-    </ul>
-    <h2>Teammate D Name (delete if you team only has 3 students) </h2>
+    <h2>Darrell Branch</h2>
     <ul>
-      <li>Summer Feelz: </li>
-      <li>Aspirational Hobby: </li>
-      <li>Ask me About: </li>
+      <li>Summer Feelz: <b><em>MOTIVATED</em></b></li>
```
Comment (NickChapman) [inline src/main/webapp/aboutus.html:43]: Nice use of the semantic markup!

File: src/main/webapp/aboutus.html [hunk 3]
```diff
@@ -41,17 +38,11 @@ <h2>Ivy Zhang</h2>
...
+      <li>Aspirational Hobby: Indie Game Dev + Foodie!</li>
+      <li>Ask me About: Basketball</li>
     </ul>
   </body>
-</html>
+</html>
```
Comment (NickChapman) [inline src/main/webapp/aboutus.html]: Please add a newline at the end of the file so that the little github "missing newline" thingy goes away :)

**Review by NickChapman — CHANGES_REQUESTED** [review]

**Review by Ivyzhang1998 — APPROVED** [review]

File: src/main/webapp/aboutus.html (hunk 1 above)
Comment (NickChapman) [reply src/main/webapp/aboutus.html:24]: Whoops! Thanks for taking care of that

**Review by NickChapman — APPROVED** [review]
Looks good. Thanks for taking care of our tag mismatches!

**Pull request merged**